app/src/main/cpp/skyline/services/am/storage/VectorIStorage.h

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <services/serviceman.h>
+#include "IStorage.h"
+
+namespace skyline::service::am {
+    /**
+     * @brief VectorIStorage is an IStorage backed by a vector<u8>
```

Comment:
It should just be `...by a vector`, it's rather pointless to specify `vector<u8>`.